### DISCUSSION TITLE
关于自定义训练识别模型精度acc始终在81%，无法提升？

### DISCUSSION THREAD

**Discussion by Eason-shu** [PaddlePaddle/PaddleOCR#13681] [Q&A] [answered]
原有的模型在一定的环境下，容易把一些字识别错，经过训练把原来一些识别错误的文字纠正了，但是就是ACC始终上不去

**Comment by Eason-shu** [comment]
![PixPin_2024-08-16_12-12-39](https://github.com/user-attachments/assets/a4e6ec26-2f6c-4d65-bed2-6fe80149106d)

**Reply by Eason-shu** [reply]
![PixPin_2024-08-16_12-13-00](https://github.com/user-attachments/assets/bb0277c0-17a1-43d4-ab73-d199346468f2)

**Reply by Eason-shu** [reply]
经过我的训练能够正确的纠正错误，但是就评估模型的时候acc始终在81

**Reply by GreatV** [reply, marked as answer]
试试调一调学习率，增加点数据量。

**Reply by Eason-shu** [reply]
好的，我在试试
